pkgs/applications/office/clickup/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, stdenv, fetchurl, appimageTools, makeWrapper, undmg, unzip }:
+
+let
+  pname = "clickup";
+  name = "clickup-desktop";
+  appname = "ClickUp";
+  version = "3.1.2";
+
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
+
+  src = fetchurl {
+    url = {
+      x86_64-linux = "https://desktop.clickup.com/linux/${appname}-{version}.AppImage";
+      aarch64-linux = "https://desktop.clickup.com/linux/${appname}-{version}.AppImage";
```

Comment:
Does it really work on aarch64-linux?

Reply:
No, you're right. The AppImage does not work on ARM devices. Fixed and pushed.